code/lib/cli/src/initiate.ts

```diff
@@ -313,22 +313,29 @@ async function doInitiate(options: CommandOptions, pkg: PackageJson): Promise<vo
   }
   done();
 
-  const storybookInstalled = isStorybookInstalled(packageJson, options.force);
+  const storybookInstatiated = isStorybookInstatiated();
 
-  if (storybookInstalled && projectType !== ProjectType.ANGULAR) {
+  if (storybookInstatiated && projectType !== ProjectType.ANGULAR) {
     logger.log();
-    paddedLog('There seems to be a Storybook already available in this project.');
-    paddedLog('Apply following command to force:\n');
-    codeLog(['sb init [options] -f']);
-
-    // Add a new line for the clear visibility.
+    const { force } = await prompts([
+      {
+        type: 'confirm',
+        name: 'force',
+        message:
+          'We found a .storybook config directory in your project. Therefore we assume that Storybook is already instantiated for your project. Do you still want to continue and force the initialization?',
+      },
+    ]);
     logger.log();
-    throw new HandledError(`Angular project already installed`);
+
+    if (force) {
+      // eslint-disable-next-line no-param-reassign
+      options.force = true;
+    }
```

Comment:
I believe if force is false, then the user doesn't want to proceed so we do `process.exit(0)`, else it will still install Storybook, right? Not sure if we should add a message or just exit the process though